<ui version="4.0" >
 <class>udp</class>
 <widget class="QWidget" name="udp" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>217</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label_35" >
       <property name="text" >
        <string>Source Port</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="leUdpSrcPort" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_38" >
       <property name="text" >
        <string>Destination Port</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="leUdpDstPort" />
     </item>
     <item row="2" column="0" >
      <widget class="QCheckBox" name="cbUdpLengthOverride" >
       <property name="text" >
        <string>Override Length</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLineEdit" name="leUdpLength" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0" >
      <widget class="QCheckBox" name="cbUdpCksumOverride" >
       <property name="text" >
        <string>Override Checksum</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QLineEdit" name="leUdpCksum" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="inputMask" >
        <string>HH HH; </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
